src/main/java/com/mixpanel/android/mpmetrics/AnalyticsMessages.java

```diff
@@ -225,7 +228,7 @@ public void handleMessage(Message msg) {
                 }
 
                 try {
-                    int queueDepth = -1;
+                    int returnCode = MPDbAdapter.DBUpdateError;
```

Comment:
We might want to add a code for "Nothing happened" or "undefined" and use it for the default value rather than defaulting to UPDATE_ERROR